src/shaders/shaders.vr.fragment.js

```diff
@@ -56,11 +56,46 @@ void getIntensity(in vec3 dataCoordinates, out float intensity, out vec3 gradien
   intensity = ( intensity - windowMin ) / uWindowCenterWidth[1];
 }
 
+void phongShading(in vec3 lightOrigin, in vec3 rayOrigin, in vec3 currentPosition, in vec3 gradient, in vec4 colorSample, out vec3 litColor){
+  vec3 normal = -normalize(gradient);
```

Comment:
nothing to here anymore  since gradient will already well oriented after suggestion from the PR thread